simplq/src/components/pages/Admin/verified-24px.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="30" height="31" viewBox="0 0 30 31"><defs><style>.a{fill:none;}.b{fill:#fff;}</style></defs><g transform="translate(0.32 0.419)"><g transform="translate(-0.32 -0.419)"><rect class="a" width="30" height="31"/></g><g transform="translate(1.181 1.876)"><path class="b" d="M27.888,15.079l-2.982-3.608L25.321,6.7l-4.412-1.06L18.6,1.5,14.444,3.388,10.288,1.5,7.979,5.625,3.567,6.673l.416,4.785L1,15.079l2.982,3.608-.416,4.785,4.412,1.06,2.31,4.125,4.155-1.9L18.6,28.645l2.31-4.125,4.412-1.06-.416-4.772Zm-15.778,6.1L7.465,16.256l1.809-1.914,2.835,3.013,7.15-7.591,1.809,1.914Z" transform="translate(-1 -1.5)"/></g></g></svg>
```

Comment:
move this svg into `simplq/public/images/`

https://github.com/SimplQ/simplQ-frontend/tree/new-frontend/simplq/public/images